libs/utils/trace_analysis.py

```diff
@@ -1303,3 +1303,38 @@ def plotClusterFrequencyResidency(self, clusters=None,
 
         self._plotFrequencyResidency(residencies, 'cluster', xmax, pct, active)
 
+    def getContextSwitchCount(self):
```

Comment:
That's true... still a more abstract and generic API can be useful sometimes.
However, agree we can add it later...